webapp/hooks/useSyncHistory/index.ts

```diff
@@ -299,7 +299,7 @@ export const useSyncHistory = function (l2ChainId: Chain['id']) {
       addWithdrawalToTunnelHistory: (withdrawal: WithdrawTunnelOperation) =>
         dispatch({ payload: withdrawal, type: 'add-withdraw' }),
       deposits: history.deposits
-        .filter(d => featureFlags.btcTunnelEnabled || d.chainId !== bitcoin.id)
+        .filter(d => d.chainId !== bitcoin.id)
```

Comment:
I think actually here, the correct thing should be removing the filter altogether

```suggestion
        .filter(d => d.chainId !== bitcoin.id)
```

if `featureFlags.btcTunnelEnabled` was enabled, the filter would always be true => the same as the filter not being there. If the feature was disabled, then we would filter out `bitcoin` deposits so they were not visible to users.

Reply:
removed